test/Interpreter/struct_extra_inhabitants.swift

```diff
@@ -167,5 +190,10 @@ tests.test("types that have extra inhabitants") {
   expectHasExtraInhabitant(GenericFullHouse<UnsafeRawPointer, UnsafeRawPointer>.self, nil: nil)
 }
 
+tests.test("types that have more than one extra inhabitant") {
+  expectHasAtLeastThreeExtraInhabitants(StringAlike64.self, nil: nil, someNil: .some(nil), someSomeNil: .some(.some(nil)))
+  expectHasAtLeastThreeExtraInhabitants(StringAlike32.self, nil: nil, someNil: .some(nil), someSomeNil: .some(.some(nil)))
```

Comment:
It's probably also worth checking that a *non-nil* object doesn't have the same representation. In particular, `_bridgeObject(taggingPayload: 0)` and `_bridgeObject(taggingPayload: 1)` are probably worth checking.